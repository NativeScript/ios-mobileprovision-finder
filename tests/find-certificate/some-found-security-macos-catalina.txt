SHA-256 hash: 1B4F0E9851971998E732078544C96B36C3D01CEDF7CAA332359D6F1D83567014
SHA-1 hash: 3B166C3B7DC4B751C9FE2AFAB9135641E388E186
-----BEGIN CERTIFICATE-----
MIIEBDCCAuygAwIBAgIIGHqpqMKWIQwwDQYJKoZIhvcNAQELBQAwYjELMAkGA1UE
BhMCVVMxEzARBgNVBAoTCkFwcGxlIEluYy4xJjAkBgNVBAsTHUFwcGxlIENlcnRp
ZmljYXRpb24gQXV0aG9yaXR5MRYwFAYDVQQDEw1BcHBsZSBSb290IENBMB4XDTEy
MDIwMTIyMTIxNVoXDTI3MDIwMTIyMTIxNVoweTEtMCsGA1UEAwwkRGV2ZWxvcGVy
IElEIENlcnRpZmljYXRpb24gQXV0aG9yaXR5MSYwJAYDVQQLDB1BcHBsZSBDZXJ0
aWZpY2F0aW9uIEF1dGhvcml0eTETMBEGA1UECgwKQXBwbGUgSW5jLjELMAkGA1UE
BhMCVVMwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQCJdk8GW5pB7qUj
KwKjX9dzP8A1sIuECj8GJH+nlT/rTw6Tr7QO0Mg+5W0Ysx/oiUe/1wkI5P9WmCkV
55SduTWjCs20wOHiYPTK7Cl4RWlpYGtfipL8niPmOsIiszFPHLrytjRZQu6wqQID
GJEEtrN4LjMfgEUNRW+7Dlpbfzrn2AjXCw4ybfuGNuRsq8QRinCEJqqfRNHxuMZ7
lBebSPcLWBa6I8WfFTl+yl3DMl8P4FJ/QOq+rAhklVvJGpzlgMofakQcbD7EsCYf
Hex7r16gaj1HqVgSMT8gdihtHRywwk4RaSaLy9bQEYLJTg/xVnTQ2QhLZniiq6yn
4tJMh1nJAgMBAAGjgaYwgaMwHQYDVR0OBBYEFFcX7aLP3HyYoRDg/L6HLSzy4xdU
MA8GA1UdEwEB/wQFMAMBAf8wHwYDVR0jBBgwFoAUK9BpR5R2Cf70a40uQKb3R01/
CF4wLgYDVR0fBCcwJTAjoCGgH4YdaHR0cDovL2NybC5hcHBsZS5jb20vcm9vdC5j
cmwwDgYDVR0PAQH/BAQDAgGGMBAGCiqGSIb3Y2QGAgYEAgUAMA0GCSqGSIb3DQEB
CwUAA4IBAQBCOXRrodzGpI83KoyzHQpEvJUsf7xZuKxh+weQkjK51L87wVA5akR0
ouxbH3Dlqt1LbBwjcS1f0cWTvu6binBlgp0W4xoQF4ktqM39DHhYSQwofzPuAHob
tHastrW7T9+oG53IGZdKC1ZnL8I+trPEgzrwd210xC4jUe6apQNvYPSlSKcGwrta
4h8fRkV+5Jf1JxC3ICJyb3LaxlB1xT0lj12jAOmfNoxIOY+zO+qQgC6VmmD0eM70
DgpTPqL6T9geroSVjTK8Vk2J6XgY4KyaQrp6RhuEoonOFOiI0ViL9q5WxCwFKkWv
C9lLqQIPNKyIx2FViUTJJ3MH7oLlTvVw
-----END CERTIFICATE-----
SHA-256 hash: 60303AE22B998861BCE3B28F33EEC1BE758A213C86C93C076DBE9F558C11C752
SHA-1 hash: 0950B6CD3D2F37EA246A1AAA20DFAADBD6FE1F75
-----BEGIN CERTIFICATE-----
MIIEIzCCAwugAwIBAgIBGTANBgkqhkiG9w0BAQUFADBiMQswCQYDVQQGEwJVUzET
MBEGA1UEChMKQXBwbGUgSW5jLjEmMCQGA1UECxMdQXBwbGUgQ2VydGlmaWNhdGlv
biBBdXRob3JpdHkxFjAUBgNVBAMTDUFwcGxlIFJvb3QgQ0EwHhcNMDgwMjE0MTg1
NjM1WhcNMTYwMjE0MTg1NjM1WjCBljELMAkGA1UEBhMCVVMxEzARBgNVBAoMCkFw
cGxlIEluYy4xLDAqBgNVBAsMI0FwcGxlIFdvcmxkd2lkZSBEZXZlbG9wZXIgUmVs
YXRpb25zMUQwQgYDVQQDDDtBcHBsZSBXb3JsZHdpZGUgRGV2ZWxvcGVyIFJlbGF0
aW9ucyBDZXJ0aWZpY2F0aW9uIEF1dGhvcml0eTCCASIwDQYJKoZIhvcNAQEBBQAD
ggEPADCCAQoCggEBAMo4VKbLVqrIJDlI6Yzu7F+4fyaRvDRTes58Y4Bhd2RepQcj
tjn+UC0VVlhwLX7EbsFKhT4v8N6EGqFXya97GP9q+hUSSRUIGayq2yoy7ZZjaFIV
PYyK7L9rGJXgA6wBfZcFZ84OhZU3au0Jtq5nzVFkn8Zc0bxXbmc1gHY2pIeBbjiP
2CsVTnsl2Fq/ToPBjdKT1RpxtWCcnTNOVfkSWAyGuBYNweV3RY1QSLorLeSUheHo
xJ3GaKWwo/xnfnC6AllLd0KRObn1zeFM78A7SIym5SFd/Wpqu6cWNWDS5q3zRinJ
6MOL6XnAamFnFbLw/eVovGJfbs+Z3e8bY/6SZasCAwEAAaOBrjCBqzAOBgNVHQ8B
Af8EBAMCAYYwDwYDVR0TAQH/BAUwAwEB/zAdBgNVHQ4EFgQUiCcXCam2GGCL7Ou6
9kdZxVJUo7cwHwYDVR0jBBgwFoAUK9BpR5R2Cf70a40uQKb3R01/CF4wNgYDVR0f
BC8wLTAroCmgJ4YlaHR0cDovL3d3dy5hcHBsZS5jb20vYXBwbGVjYS9yb290LmNy
bDAQBgoqhkiG92NkBgIBBAIFADANBgkqhkiG9w0BAQUFAAOCAQEA2jIAlsVUlNM7
gjdmfS5o1cPGuMsmjEiQzxMkakaOY9Tw0BMG3djEwTcV8jMTOSYtzi5VQOMLA6/6
EsLnDSG41YDPrCgvzi2zTq+GGQTG6VDdTClHECP8bLsbmGtIieFbnd5G2zWFNe8+
0OJYSzj07XVaH1xwHVY5EuXhDRHkiSUGvdW0FY5e0FmXkOlLgeLfGK9EdB4ZoDpH
zJEdOusjWv6lLZf3e7vWh0ZChetSPSayY6i0scqP9Mzis8hH4L+aWYP62phTKoL1
fGUuldkzXfXtZcwxN8VaBOhr4eeIA0p1npsoy0pAiGVDdd3LOiUjxZ5X+C7O0qmS
XnMuLyV1FQ==
-----END CERTIFICATE-----
SHA-256 hash: FD61A03AF4F77D870FC21E05E7E80678095C92D808CFB3B5C279EE04C74ACA13
SHA-1 hash: 4BBBD768D1A5D61FA9B7F5E9F211F054CFB2E73E
-----BEGIN CERTIFICATE-----
MIIFmTCCBIGgAwIBAgIIVggUK9gU5EgwDQYJKoZIhvcNAQEFBQAwgZYxCzAJBgNV
BAYTAlVTMRMwEQYDVQQKDApBcHBsZSBJbmMuMSwwKgYDVQQLDCNBcHBsZSBXb3Js
ZHdpZGUgRGV2ZWxvcGVyIFJlbGF0aW9uczFEMEIGA1UEAww7QXBwbGUgV29ybGR3
aWRlIERldmVsb3BlciBSZWxhdGlvbnMgQ2VydGlmaWNhdGlvbiBBdXRob3JpdHkw
HhcNMTQwMjExMTI0MzIwWhcNMTUwMjExMTI0MzIwWjCBjDEaMBgGCgmSJomT8ixk
AQEMCkxYV0RTSDZQWTYxNjA0BgNVBAMMLWlQaG9uZSBEZXZlbG9wZXI6IFRzdmV0
YW4gUmFpa292ICg5SFE4MjdGTFZLKTETMBEGA1UECwwKQ0hTUTNNM1AzNzEUMBIG
A1UECgwLVGVsZXJpayBBIEQxCzAJBgNVBAYTAkJHMIIBIjANBgkqhkiG9w0BAQEF
AAOCAQ8AMIIBCgKCAQEA1Jo4zPQnN6g26ezUQTdPQ1SLLgE/5RVGivOZHHYBkPxR
CJIrlt2o1pjfykgLE8xCo0CkNRseAhqeRVBciTtktoDbBI9WnA3DVGvWiuRi1JI5
t26BClZiYJH/Zkt42oVNGXyLjLuE40mc8oqeSS5z3KaDuF+oQ5ItRMDaTL7wBxbE
xmIkE6Qdtjwnu4TLqeGVcIDTe2JBkC+UTeVzqAOOSXSA8ufFw4vf2WS6b4CahU+0
hHVvggxjNcVQ6wH5SWnPBI0F1Fx5HKnqmrRN6d5SWQVQEsgFzfkuDmO0xPNB6Ufp
3sOujlX909C/e0zYaEXNDOQiV6iMtJKxaiAbqBFFRwIDAQABo4IB8TCCAe0wHQYD
VR0OBBYEFMVy//epz7d2qUPMoVbkuo1SQMjJMAwGA1UdEwEB/wQCMAAwHwYDVR0j
BBgwFoAUiCcXCam2GGCL7Ou69kdZxVJUo7cwggEPBgNVHSAEggEGMIIBAjCB/wYJ
KoZIhvdjZAUBMIHxMIHDBggrBgEFBQcCAjCBtgyBs1JlbGlhbmNlIG9uIHRoaXMg
Y2VydGlmaWNhdGUgYnkgYW55IHBhcnR5IGFzc3VtZXMgYWNjZXB0YW5jZSBvZiB0
aGUgdGhlbiBhcHBsaWNhYmxlIHN0YW5kYXJkIHRlcm1zIGFuZCBjb25kaXRpb25z
IG9mIHVzZSwgY2VydGlmaWNhdGUgcG9saWN5IGFuZCBjZXJ0aWZpY2F0aW9uIHBy
YWN0aWNlIHN0YXRlbWVudHMuMCkGCCsGAQUFBwIBFh1odHRwOi8vd3d3LmFwcGxl
LmNvbS9hcHBsZWNhLzBNBgNVHR8ERjBEMEKgQKA+hjxodHRwOi8vZGV2ZWxvcGVy
LmFwcGxlLmNvbS9jZXJ0aWZpY2F0aW9uYXV0aG9yaXR5L3d3ZHJjYS5jcmwwDgYD
VR0PAQH/BAQDAgeAMBYGA1UdJQEB/wQMMAoGCCsGAQUFBwMDMBMGCiqGSIb3Y2QG
AQIBAf8EAgUAMA0GCSqGSIb3DQEBBQUAA4IBAQBtKkPHM6XeOcnRSkbn6otAjK55
y+pyLo18azP8ovCaNqXQjrmCDXTLOZssEUhXd2tvlCTv/ZqJeE6oBa+6qSfqeQ6T
8SX1B/mZabSmNHTGFoezWZ1Zm/c51nFHsc/WE6muLcs4DovqjVe/MK7X5bpqf6O5
5axdm39nOz8INUY+NMTmiPixkGpI/1z0gwN4RdD0OqyxdDCL60r8uIDxxOWDDy28
wFIzGVMRBO40pPu3LYoj4E78j+iAPqFxP2/5G0whe5jmJa7pJmNro5F9COEL4HYT
z6fxSZa2A5x2sU+W8E7nz/+1vQQ25D/VBFgKponDEbzts2EcKHah3gJbsfqF
-----END CERTIFICATE-----
SHA-256 hash: A4E624D686E03ED2767C0ABD85C14426B0B1157D2CE81D27BB4FE4F6F01D688A
SHA-1 hash: E774DCE3E91D642344F81E00FD901B3256484CAA
-----BEGIN CERTIFICATE-----
MIIFmjCCBIKgAwIBAgIII11d0wV6k18wDQYJKoZIhvcNAQEFBQAwgZYxCzAJBgNV
BAYTAlVTMRMwEQYDVQQKDApBcHBsZSBJbmMuMSwwKgYDVQQLDCNBcHBsZSBXb3Js
ZHdpZGUgRGV2ZWxvcGVyIFJlbGF0aW9uczFEMEIGA1UEAww7QXBwbGUgV29ybGR3
aWRlIERldmVsb3BlciBSZWxhdGlvbnMgQ2VydGlmaWNhdGlvbiBBdXRob3JpdHkw
HhcNMTQwMTA4MDk0ODIwWhcNMTUwMTA4MDk0ODIwWjCBjTEaMBgGCgmSJomT8ixk
AQEMClc2NkRDOEdXWlExNzA1BgNVBAMMLmlQaG9uZSBEZXZlbG9wZXI6IEVtaWwg
TWFyYXNobGlldiAoTTU3WFVDQVhFOSkxEzARBgNVBAsMCkNIU1EzTTNQMzcxFDAS
BgNVBAoMC1RlbGVyaWsgQSBEMQswCQYDVQQGEwJCRzCCASIwDQYJKoZIhvcNAQEB
BQADggEPADCCAQoCggEBAMLlgMTFANsXvgzz4Kj5W3EiAUaj2r3wKqCaFreRcNH+
KEjOJFZN75Y0RkC9kNoE8b+RHmTHSlgPf0G4kyp/tKVB97gQmd6j+1hbsSG9efzi
HTfsFjtsNcOApfdMKyDI8/I7nh2tuM9QGwtXYRBx0Hh5F9edIu4jaYdqUu7I5zXJ
f2n+cPvj9OFpO/aD/WZB/eJ33wwRx66bBUBTvrBqBPaqrPgTUBpvDObYySPbXqaD
IgCbExFeDb21/Oh9/aMCwCtX/VHTiuahO9knlhCpXh7vVUF0qlIhngiXv37p7tDF
nJj4Tv1NDvBrydefbJlO7mQlX5REv+NyifYhLZj1ypsCAwEAAaOCAfEwggHtMB0G
A1UdDgQWBBSRmbJTXR020AWUX4os0xwPk+LEUTAMBgNVHRMBAf8EAjAAMB8GA1Ud
IwQYMBaAFIgnFwmpthhgi+zruvZHWcVSVKO3MIIBDwYDVR0gBIIBBjCCAQIwgf8G
CSqGSIb3Y2QFATCB8TCBwwYIKwYBBQUHAgIwgbYMgbNSZWxpYW5jZSBvbiB0aGlz
IGNlcnRpZmljYXRlIGJ5IGFueSBwYXJ0eSBhc3N1bWVzIGFjY2VwdGFuY2Ugb2Yg
dGhlIHRoZW4gYXBwbGljYWJsZSBzdGFuZGFyZCB0ZXJtcyBhbmQgY29uZGl0aW9u
cyBvZiB1c2UsIGNlcnRpZmljYXRlIHBvbGljeSBhbmQgY2VydGlmaWNhdGlvbiBw
cmFjdGljZSBzdGF0ZW1lbnRzLjApBggrBgEFBQcCARYdaHR0cDovL3d3dy5hcHBs
ZS5jb20vYXBwbGVjYS8wTQYDVR0fBEYwRDBCoECgPoY8aHR0cDovL2RldmVsb3Bl
ci5hcHBsZS5jb20vY2VydGlmaWNhdGlvbmF1dGhvcml0eS93d2RyY2EuY3JsMA4G
A1UdDwEB/wQEAwIHgDAWBgNVHSUBAf8EDDAKBggrBgEFBQcDAzATBgoqhkiG92Nk
BgECAQH/BAIFADANBgkqhkiG9w0BAQUFAAOCAQEAZlSj7LP1CmkXGBnUhAwEqjZM
gRfM6sfZ6EiEqUhN7yYbzNPyUO6tgTEjxtOXfb59CfHtBSkrFu7xps2Xl2XJLqfg
GmyAURHo40bXo5GpDV9mH9jsTSSy8fIQMllmDQfx+yq715FLp+k1NF5JvMVJ3L7h
MTsQAtCTNL8IFHZUVJM6SOAmyMNKndwobquQ12ndTRi6r/LydV5dAVLM44RAbMUf
Knd3WGKKS2xYfgSWiOIQjRA3+LUffYJ1Zm+QZ4zP0auJkE0TMGcOZNXWmKjv8Zmb
om/IfnrhFm4c/a+qAeInlIJ+c2ds5qvTGaxMAkMNmapD9cn2UgZhv1yphN73qQ==
-----END CERTIFICATE-----
