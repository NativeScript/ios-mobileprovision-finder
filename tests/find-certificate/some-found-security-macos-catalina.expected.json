[
	{
		"hash": "3B166C3B7DC4B751C9FE2AFAB9135641E388E186",
		"pem": "MIIEBDCCAuygAwIBAgIIGHqpqMKWIQwwDQYJKoZIhvcNAQELBQAwYjELMAkGA1UE\nBhMCVVMxEzARBgNVBAoTCkFwcGxlIEluYy4xJjAkBgNVBAsTHUFwcGxlIENlcnRp\nZmljYXRpb24gQXV0aG9yaXR5MRYwFAYDVQQDEw1BcHBsZSBSb290IENBMB4XDTEy\nMDIwMTIyMTIxNVoXDTI3MDIwMTIyMTIxNVoweTEtMCsGA1UEAwwkRGV2ZWxvcGVy\nIElEIENlcnRpZmljYXRpb24gQXV0aG9yaXR5MSYwJAYDVQQLDB1BcHBsZSBDZXJ0\naWZpY2F0aW9uIEF1dGhvcml0eTETMBEGA1UECgwKQXBwbGUgSW5jLjELMAkGA1UE\nBhMCVVMwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQCJdk8GW5pB7qUj\nKwKjX9dzP8A1sIuECj8GJH+nlT/rTw6Tr7QO0Mg+5W0Ysx/oiUe/1wkI5P9WmCkV\n55SduTWjCs20wOHiYPTK7Cl4RWlpYGtfipL8niPmOsIiszFPHLrytjRZQu6wqQID\nGJEEtrN4LjMfgEUNRW+7Dlpbfzrn2AjXCw4ybfuGNuRsq8QRinCEJqqfRNHxuMZ7\nlBebSPcLWBa6I8WfFTl+yl3DMl8P4FJ/QOq+rAhklVvJGpzlgMofakQcbD7EsCYf\nHex7r16gaj1HqVgSMT8gdihtHRywwk4RaSaLy9bQEYLJTg/xVnTQ2QhLZniiq6yn\n4tJMh1nJAgMBAAGjgaYwgaMwHQYDVR0OBBYEFFcX7aLP3HyYoRDg/L6HLSzy4xdU\nMA8GA1UdEwEB/wQFMAMBAf8wHwYDVR0jBBgwFoAUK9BpR5R2Cf70a40uQKb3R01/\nCF4wLgYDVR0fBCcwJTAjoCGgH4YdaHR0cDovL2NybC5hcHBsZS5jb20vcm9vdC5j\ncmwwDgYDVR0PAQH/BAQDAgGGMBAGCiqGSIb3Y2QGAgYEAgUAMA0GCSqGSIb3DQEB\nCwUAA4IBAQBCOXRrodzGpI83KoyzHQpEvJUsf7xZuKxh+weQkjK51L87wVA5akR0\nouxbH3Dlqt1LbBwjcS1f0cWTvu6binBlgp0W4xoQF4ktqM39DHhYSQwofzPuAHob\ntHastrW7T9+oG53IGZdKC1ZnL8I+trPEgzrwd210xC4jUe6apQNvYPSlSKcGwrta\n4h8fRkV+5Jf1JxC3ICJyb3LaxlB1xT0lj12jAOmfNoxIOY+zO+qQgC6VmmD0eM70\nDgpTPqL6T9geroSVjTK8Vk2J6XgY4KyaQrp6RhuEoonOFOiI0ViL9q5WxCwFKkWv\nC9lLqQIPNKyIx2FViUTJJ3MH7oLlTvVw"
	},
	{
		"hash": "0950B6CD3D2F37EA246A1AAA20DFAADBD6FE1F75",
		"pem": "MIIEIzCCAwugAwIBAgIBGTANBgkqhkiG9w0BAQUFADBiMQswCQYDVQQGEwJVUzET\nMBEGA1UEChMKQXBwbGUgSW5jLjEmMCQGA1UECxMdQXBwbGUgQ2VydGlmaWNhdGlv\nbiBBdXRob3JpdHkxFjAUBgNVBAMTDUFwcGxlIFJvb3QgQ0EwHhcNMDgwMjE0MTg1\nNjM1WhcNMTYwMjE0MTg1NjM1WjCBljELMAkGA1UEBhMCVVMxEzARBgNVBAoMCkFw\ncGxlIEluYy4xLDAqBgNVBAsMI0FwcGxlIFdvcmxkd2lkZSBEZXZlbG9wZXIgUmVs\nYXRpb25zMUQwQgYDVQQDDDtBcHBsZSBXb3JsZHdpZGUgRGV2ZWxvcGVyIFJlbGF0\naW9ucyBDZXJ0aWZpY2F0aW9uIEF1dGhvcml0eTCCASIwDQYJKoZIhvcNAQEBBQAD\nggEPADCCAQoCggEBAMo4VKbLVqrIJDlI6Yzu7F+4fyaRvDRTes58Y4Bhd2RepQcj\ntjn+UC0VVlhwLX7EbsFKhT4v8N6EGqFXya97GP9q+hUSSRUIGayq2yoy7ZZjaFIV\nPYyK7L9rGJXgA6wBfZcFZ84OhZU3au0Jtq5nzVFkn8Zc0bxXbmc1gHY2pIeBbjiP\n2CsVTnsl2Fq/ToPBjdKT1RpxtWCcnTNOVfkSWAyGuBYNweV3RY1QSLorLeSUheHo\nxJ3GaKWwo/xnfnC6AllLd0KRObn1zeFM78A7SIym5SFd/Wpqu6cWNWDS5q3zRinJ\n6MOL6XnAamFnFbLw/eVovGJfbs+Z3e8bY/6SZasCAwEAAaOBrjCBqzAOBgNVHQ8B\nAf8EBAMCAYYwDwYDVR0TAQH/BAUwAwEB/zAdBgNVHQ4EFgQUiCcXCam2GGCL7Ou6\n9kdZxVJUo7cwHwYDVR0jBBgwFoAUK9BpR5R2Cf70a40uQKb3R01/CF4wNgYDVR0f\nBC8wLTAroCmgJ4YlaHR0cDovL3d3dy5hcHBsZS5jb20vYXBwbGVjYS9yb290LmNy\nbDAQBgoqhkiG92NkBgIBBAIFADANBgkqhkiG9w0BAQUFAAOCAQEA2jIAlsVUlNM7\ngjdmfS5o1cPGuMsmjEiQzxMkakaOY9Tw0BMG3djEwTcV8jMTOSYtzi5VQOMLA6/6\nEsLnDSG41YDPrCgvzi2zTq+GGQTG6VDdTClHECP8bLsbmGtIieFbnd5G2zWFNe8+\n0OJYSzj07XVaH1xwHVY5EuXhDRHkiSUGvdW0FY5e0FmXkOlLgeLfGK9EdB4ZoDpH\nzJEdOusjWv6lLZf3e7vWh0ZChetSPSayY6i0scqP9Mzis8hH4L+aWYP62phTKoL1\nfGUuldkzXfXtZcwxN8VaBOhr4eeIA0p1npsoy0pAiGVDdd3LOiUjxZ5X+C7O0qmS\nXnMuLyV1FQ=="
	},
	{
		"hash": "4BBBD768D1A5D61FA9B7F5E9F211F054CFB2E73E",
		"pem": "MIIFmTCCBIGgAwIBAgIIVggUK9gU5EgwDQYJKoZIhvcNAQEFBQAwgZYxCzAJBgNV\nBAYTAlVTMRMwEQYDVQQKDApBcHBsZSBJbmMuMSwwKgYDVQQLDCNBcHBsZSBXb3Js\nZHdpZGUgRGV2ZWxvcGVyIFJlbGF0aW9uczFEMEIGA1UEAww7QXBwbGUgV29ybGR3\naWRlIERldmVsb3BlciBSZWxhdGlvbnMgQ2VydGlmaWNhdGlvbiBBdXRob3JpdHkw\nHhcNMTQwMjExMTI0MzIwWhcNMTUwMjExMTI0MzIwWjCBjDEaMBgGCgmSJomT8ixk\nAQEMCkxYV0RTSDZQWTYxNjA0BgNVBAMMLWlQaG9uZSBEZXZlbG9wZXI6IFRzdmV0\nYW4gUmFpa292ICg5SFE4MjdGTFZLKTETMBEGA1UECwwKQ0hTUTNNM1AzNzEUMBIG\nA1UECgwLVGVsZXJpayBBIEQxCzAJBgNVBAYTAkJHMIIBIjANBgkqhkiG9w0BAQEF\nAAOCAQ8AMIIBCgKCAQEA1Jo4zPQnN6g26ezUQTdPQ1SLLgE/5RVGivOZHHYBkPxR\nCJIrlt2o1pjfykgLE8xCo0CkNRseAhqeRVBciTtktoDbBI9WnA3DVGvWiuRi1JI5\nt26BClZiYJH/Zkt42oVNGXyLjLuE40mc8oqeSS5z3KaDuF+oQ5ItRMDaTL7wBxbE\nxmIkE6Qdtjwnu4TLqeGVcIDTe2JBkC+UTeVzqAOOSXSA8ufFw4vf2WS6b4CahU+0\nhHVvggxjNcVQ6wH5SWnPBI0F1Fx5HKnqmrRN6d5SWQVQEsgFzfkuDmO0xPNB6Ufp\n3sOujlX909C/e0zYaEXNDOQiV6iMtJKxaiAbqBFFRwIDAQABo4IB8TCCAe0wHQYD\nVR0OBBYEFMVy//epz7d2qUPMoVbkuo1SQMjJMAwGA1UdEwEB/wQCMAAwHwYDVR0j\nBBgwFoAUiCcXCam2GGCL7Ou69kdZxVJUo7cwggEPBgNVHSAEggEGMIIBAjCB/wYJ\nKoZIhvdjZAUBMIHxMIHDBggrBgEFBQcCAjCBtgyBs1JlbGlhbmNlIG9uIHRoaXMg\nY2VydGlmaWNhdGUgYnkgYW55IHBhcnR5IGFzc3VtZXMgYWNjZXB0YW5jZSBvZiB0\naGUgdGhlbiBhcHBsaWNhYmxlIHN0YW5kYXJkIHRlcm1zIGFuZCBjb25kaXRpb25z\nIG9mIHVzZSwgY2VydGlmaWNhdGUgcG9saWN5IGFuZCBjZXJ0aWZpY2F0aW9uIHBy\nYWN0aWNlIHN0YXRlbWVudHMuMCkGCCsGAQUFBwIBFh1odHRwOi8vd3d3LmFwcGxl\nLmNvbS9hcHBsZWNhLzBNBgNVHR8ERjBEMEKgQKA+hjxodHRwOi8vZGV2ZWxvcGVy\nLmFwcGxlLmNvbS9jZXJ0aWZpY2F0aW9uYXV0aG9yaXR5L3d3ZHJjYS5jcmwwDgYD\nVR0PAQH/BAQDAgeAMBYGA1UdJQEB/wQMMAoGCCsGAQUFBwMDMBMGCiqGSIb3Y2QG\nAQIBAf8EAgUAMA0GCSqGSIb3DQEBBQUAA4IBAQBtKkPHM6XeOcnRSkbn6otAjK55\ny+pyLo18azP8ovCaNqXQjrmCDXTLOZssEUhXd2tvlCTv/ZqJeE6oBa+6qSfqeQ6T\n8SX1B/mZabSmNHTGFoezWZ1Zm/c51nFHsc/WE6muLcs4DovqjVe/MK7X5bpqf6O5\n5axdm39nOz8INUY+NMTmiPixkGpI/1z0gwN4RdD0OqyxdDCL60r8uIDxxOWDDy28\nwFIzGVMRBO40pPu3LYoj4E78j+iAPqFxP2/5G0whe5jmJa7pJmNro5F9COEL4HYT\nz6fxSZa2A5x2sU+W8E7nz/+1vQQ25D/VBFgKponDEbzts2EcKHah3gJbsfqF"
	},
	{
		"hash": "E774DCE3E91D642344F81E00FD901B3256484CAA",
		"pem": "MIIFmjCCBIKgAwIBAgIII11d0wV6k18wDQYJKoZIhvcNAQEFBQAwgZYxCzAJBgNV\nBAYTAlVTMRMwEQYDVQQKDApBcHBsZSBJbmMuMSwwKgYDVQQLDCNBcHBsZSBXb3Js\nZHdpZGUgRGV2ZWxvcGVyIFJlbGF0aW9uczFEMEIGA1UEAww7QXBwbGUgV29ybGR3\naWRlIERldmVsb3BlciBSZWxhdGlvbnMgQ2VydGlmaWNhdGlvbiBBdXRob3JpdHkw\nHhcNMTQwMTA4MDk0ODIwWhcNMTUwMTA4MDk0ODIwWjCBjTEaMBgGCgmSJomT8ixk\nAQEMClc2NkRDOEdXWlExNzA1BgNVBAMMLmlQaG9uZSBEZXZlbG9wZXI6IEVtaWwg\nTWFyYXNobGlldiAoTTU3WFVDQVhFOSkxEzARBgNVBAsMCkNIU1EzTTNQMzcxFDAS\nBgNVBAoMC1RlbGVyaWsgQSBEMQswCQYDVQQGEwJCRzCCASIwDQYJKoZIhvcNAQEB\nBQADggEPADCCAQoCggEBAMLlgMTFANsXvgzz4Kj5W3EiAUaj2r3wKqCaFreRcNH+\nKEjOJFZN75Y0RkC9kNoE8b+RHmTHSlgPf0G4kyp/tKVB97gQmd6j+1hbsSG9efzi\nHTfsFjtsNcOApfdMKyDI8/I7nh2tuM9QGwtXYRBx0Hh5F9edIu4jaYdqUu7I5zXJ\nf2n+cPvj9OFpO/aD/WZB/eJ33wwRx66bBUBTvrBqBPaqrPgTUBpvDObYySPbXqaD\nIgCbExFeDb21/Oh9/aMCwCtX/VHTiuahO9knlhCpXh7vVUF0qlIhngiXv37p7tDF\nnJj4Tv1NDvBrydefbJlO7mQlX5REv+NyifYhLZj1ypsCAwEAAaOCAfEwggHtMB0G\nA1UdDgQWBBSRmbJTXR020AWUX4os0xwPk+LEUTAMBgNVHRMBAf8EAjAAMB8GA1Ud\nIwQYMBaAFIgnFwmpthhgi+zruvZHWcVSVKO3MIIBDwYDVR0gBIIBBjCCAQIwgf8G\nCSqGSIb3Y2QFATCB8TCBwwYIKwYBBQUHAgIwgbYMgbNSZWxpYW5jZSBvbiB0aGlz\nIGNlcnRpZmljYXRlIGJ5IGFueSBwYXJ0eSBhc3N1bWVzIGFjY2VwdGFuY2Ugb2Yg\ndGhlIHRoZW4gYXBwbGljYWJsZSBzdGFuZGFyZCB0ZXJtcyBhbmQgY29uZGl0aW9u\ncyBvZiB1c2UsIGNlcnRpZmljYXRlIHBvbGljeSBhbmQgY2VydGlmaWNhdGlvbiBw\ncmFjdGljZSBzdGF0ZW1lbnRzLjApBggrBgEFBQcCARYdaHR0cDovL3d3dy5hcHBs\nZS5jb20vYXBwbGVjYS8wTQYDVR0fBEYwRDBCoECgPoY8aHR0cDovL2RldmVsb3Bl\nci5hcHBsZS5jb20vY2VydGlmaWNhdGlvbmF1dGhvcml0eS93d2RyY2EuY3JsMA4G\nA1UdDwEB/wQEAwIHgDAWBgNVHSUBAf8EDDAKBggrBgEFBQcDAzATBgoqhkiG92Nk\nBgECAQH/BAIFADANBgkqhkiG9w0BAQUFAAOCAQEAZlSj7LP1CmkXGBnUhAwEqjZM\ngRfM6sfZ6EiEqUhN7yYbzNPyUO6tgTEjxtOXfb59CfHtBSkrFu7xps2Xl2XJLqfg\nGmyAURHo40bXo5GpDV9mH9jsTSSy8fIQMllmDQfx+yq715FLp+k1NF5JvMVJ3L7h\nMTsQAtCTNL8IFHZUVJM6SOAmyMNKndwobquQ12ndTRi6r/LydV5dAVLM44RAbMUf\nKnd3WGKKS2xYfgSWiOIQjRA3+LUffYJ1Zm+QZ4zP0auJkE0TMGcOZNXWmKjv8Zmb\nom/IfnrhFm4c/a+qAeInlIJ+c2ds5qvTGaxMAkMNmapD9cn2UgZhv1yphN73qQ=="
	}
]
